docs/07.Reference/7.02.Filters.md

```diff
@@ -2071,6 +2071,10 @@ package, and extra functions defined by Easegress:
   also an object, its type must also be `map[string]interface{}`.
 * **jsonEscape**: escape a string so that it can be used as the key or value
   in JSON text.
+* **urlQueryEscape**: escapes the string so that it can be safely placed inside
+  a URL query, equivalent to the `urlquery` template function.
```

Comment:
Hi, what's the difference between `urlQueryEscape` and `urlquery`? 
And could you please tell me why we need to add `urlQueryEscape` if there are already `urlquery`?

Reply:
urlquery is a built-in template function in the Go template package, and it has the same implementation as urlQueryEscape. The reason for adding both urlQueryEscape and urlQueryUnescape functions simultaneously serves two purposes:

1. To provide a clearer expression of functionality, as these functions are implemented using the queryEscape and queryUnescape methods from the url package.
2. To maintain consistency within Easegress by using symmetric function names.